tests/fails/002_interprocedural.rs

```diff
@@ -0,0 +1,21 @@
+#![feature(register_tool)]
+#![register_tool(taint)]
+
+fn main() {
+    let val = get_input();
+    output(val); //~ ERROR
```

Comment:
Maybe it would be good to annotate the exact error we expect.

Reply:
I agree, but I also think it would make sense to do a large overhaul of the tests in a separate PR, because this one is getting a little large, and I want to make more changes to the tests anyway. 👼🏼

Reply:
Ehh, fixed them anyway :D